packages/pn-personafisica-webapp/src/api/interceptors.ts

```diff
@@ -13,4 +28,36 @@ export const setUpInterceptor = (store: EnhancedStore) => {
     },
     (error) => Promise.reject(error)
   );
-};
\ No newline at end of file
+
+  axiosResponseInterceptor = apiClient.interceptors.response.use(
+    (response) => {
+      if (response.config?.url === '/delivery/notifications/received/DKRU-XUDK-UERQ-202308-X-1') {
```

Comment:
create also the case fot CANCELLATION_IN_PROGRESS

Reply:
do it